### ISSUE TITLE
ModuleNotFoundError: No module named 'entities'

### ISSUE THREAD

**Issue by shandygunawan** [AlexandreSenpai/NHentai-API#8]
Hi, I hope you are doing well in this pandemic situation.
I tried using your nhentai api and got this error message when trying to import the package: `from NHentai import NHentai` .
This error persists when I'm trying to install it in Google Colab (Python 3.6.9) or my local virtual environment (Python 3.8.0).
Thanks in advance.

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-3-1dc4a5a8e803> in <module>()
----> 1 from NHentai import NHentai

/usr/local/lib/python3.6/dist-packages/NHentai/__init__.py in <module>()
      6 import requests
      7 
----> 8 from entities.doujin import Doujin, DoujinThumbnail
      9 from entities.page import HomePage, SearchPage, TagListPage, GroupListPage, CharacterListPage, ArtistListPage
     10 from entities.links import CharacterLink

ModuleNotFoundError: No module named 'entities'
```

**Comment by AlexandreSenpai** [comment]
Sorry for my late reply. 
I'm going to fix this issue as soon as possible.
Thanks for the report.

**Comment by AlexandreSenpai** [comment]
Fixed on version 0.0.5
and
NHentai-API==0.0.14 pypi module
